<launch>
    <arg name="labels_file" default="$(find milestone3)/scripts/labels"/>
    <arg name="coco_file" default="$(find milestone3)/scripts/perception/coco.names"/>
    <arg name="cfg_file" default="$(find milestone3)/scripts/perception/yolov3-tiny.cfg"/>
    <arg name="weights_file" default="$(find milestone3)/scripts/perception/yolov3-tiny.weights"/>


    <node pkg="milestone3" type="undistort_img.py" name="undistort_img">
        <param name="mtx" value="$(find milestone3)/scripts/perception/mtx.npy"/>
        <param name="dist" value="$(find milestone3)/scripts/perception/dist.npy"/>
    </node>
    <node pkg="milestone3" type="deep_learning" name="deep_learning">
        <param name="coco" value="$(arg coco_file)"/>
        <param name="cfg" value="$(arg cfg_file)"/>
        <param name="weights" value="$(arg weights_file)"/>
    </node>
    <!--node pkg="milestone3" type="navi" name="navi">
        <param name="labels" value="$(arg labels_file)"/>
    </node-->

    <node pkg="milestone3" type="sign_pose" name="sign_detected"></node>

    <node pkg="milestone3" type="sign_tf" name="sign_tf">
        <param name="labels" value="$(arg labels_file)"/>
    </node>

   
</launch>
